fuseimg/data/ops/shape_ops.py

```diff
@@ -95,20 +95,23 @@ def pad_image(self,inner_image: np.ndarray, padding: Tuple[float, float], resize
         pads image to requested size ,
         pads both side equally by the same input padding size (left = right = padding[1] , up = down= padding[0] )  ,
         padding default value is zero or minimum value in normalized target range
-        :param inner_image: image of shape [H, W, C]
+        :param inner_image: image of shape [H, W, C] of type numpy float32
         :param padding: required padding [x,y]
         :param resize_to: original requested resolution
         :param normalized_target_range: requested normalized image pixels range
         :param number_of_channels: number of color channels in the image
         :return: padded image
         """
-        inner_image = inner_image.astype('float32')
-        # "Pad" around inner image
+        # "Pad" around inner image with background value 
         inner_image_height, inner_image_width = inner_image.shape[0], inner_image.shape[1]
-        inner_image[0:inner_image_height, 0] = 0
-        inner_image[0:inner_image_height, inner_image_width - 1] = 0
-        inner_image[0, 0:inner_image_width] = 0
-        inner_image[inner_image_height - 1, 0:inner_image_width] = 0
+        # pad first row
```

Comment:
Why do you need to pad here if you apply padding 116, is it because you are using reflect method?